src/api/app/views/webui2/webui/request/_issues_table.html.haml

```diff
@@ -0,0 +1,23 @@
+%p
+  %b
+    Mentioned Issues (#{issues.length})
+%table
+  %tbody
+    - issues.each do |long_name, issue|
+      - if issue[:owner] && issue[:state]
+        %tr
+          %td
+            = link_to(long_name, issue[:url], target: '_blank', rel: 'noopener')
```

Comment:
[nitpick] As said before, I would prefer the user to be :free: to decide where to open the links.... :sob: